docs/resource_management/RESOURCE_SCALING.md

```diff
@@ -0,0 +1,61 @@
+# Product scaling
+For optimum performance and stability the appropriate resource `request` for each pod, and the number of pods in the product cluster should be carefully considered. Kubernetes provides a means for horizontally and vertically scaling the pods within a cluster, these approaches are described below.
+
+## Horizontal scaling
+The Helm charts provision one `StatefulSet` by default. In order to horizontally scale up or down the cluster `kubectl scale` can be used at runtime to provision a multi-node Data Center cluster, with no further configuration required (although note that the Ingress must support cookie-based session affinity in order for the products to work correctly in a multi-node configuration). Here is the syntax for scaling up/down the Data Center cluster:
+```
+kubectl scale statefulsets <statefulsetset-name> --replicas=n
+```
+For details on the `cpu` and `memory` requirements see [Resource requests and limits](#Resource-requests-and-limits) 
+
+## Vertical scaling
+The resource `requests` and `limits` for a `StatefulSet` can be defined either post product deployment or for deployments that are already running within the Kubernetes cluster. Take note that vertical scaling will result in the pod being re-created with the updated values
+
+### Prior to deployment
+Before performing a helm install update the appropriate products `values.yaml` `container` stanza with the desired `requests` and `limits` values i.e. 
+```yaml
+ container: 
+  limits:
+    cpu: "4"
+    memory: "4G"
+  requests:
+    cpu: "2"
+    memory: "2G"
+```
+
+### Post deployment
+For existing deployments the `requests` and `limits` values can be dynamically updated either declaratively or intrinsically 
+
+#### Declaratively
+This the preferred approach as it keeps the state of the cluster, and the helm charts themselves in sync.
+1. Update `values.yaml` appropriately
+2. Apply the patch:
+```shell
+helm upgrade <release> <chart> -f <values file>
+```
+
+#### Intrinsically
+Using `kubectl edit` on the appropriate `StatefulSet` the respective `cpu` and `memory` values can be modified. Saving the changes will then result in the existing product pod(s) being re-provisioned with the updated values.
+
+
+## Resource requests and limits
+To ensure that Kubernetes appropriately schedules resources, the respective product `values.yaml` are configured with default `cpu` and `memory` [resource request values](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
```

Comment:
This is similar to the previous section, we also mention the URL for kubernetes documentation for the first time here, it should be linked early in the document.

Reply:
the respective product `values.yaml` is configured

Reply:
I dont think mentioning this URL here for the first time is a bad thing. The section as a whole (`Resource requests and limits`) is meant to provide additional colour/background to scaling in general. Im thinking it might be better  if this is split out into its own readme

Reply:
Ive made this split which makes things clearer.